synapseclient/api/form_services.py

```diff
@@ -0,0 +1,203 @@
+import json
```

Comment:
Are these functions enough for Phil?

Reply:
I think so! I have asked Phil to take a look before he goes on to vacation next week

Reply:
After a private conversation, no need to shift it. Lets proceed with OOP as it may be a better developer experience to work with these models